2025-02-25T03:23:24.2131690Z ##[group]Run pytest tardis --tardis-regression-data=/Users/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append -m "not continuum"
2025-02-25T03:23:24.2132840Z [36;1mpytest tardis --tardis-regression-data=/Users/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append -m "not continuum"[0m
2025-02-25T03:23:24.2237330Z shell: /bin/bash -l {0}
2025-02-25T03:23:24.2237650Z env:
2025-02-25T03:23:24.2237920Z   CACHE_NUMBER: 0
2025-02-25T03:23:24.2238590Z   PYTEST_FLAGS: --tardis-regression-data=/Users/runner/work/tardis/tardis/tardis-regression-data --cov=tardis --cov-report=xml --cov-report=html --cov-append
2025-02-25T03:23:24.2239220Z   CODECOV_TOKEN: 
2025-02-25T03:23:24.2239540Z   MAMBA_ROOT_PREFIX: /Users/runner/micromamba
2025-02-25T03:23:24.2239860Z   MAMBA_EXE: /Users/runner/micromamba-bin/micromamba
2025-02-25T03:23:24.2240290Z   CONDARC: /Users/runner/work/_temp/setup-micromamba/.condarc
2025-02-25T03:23:24.2240770Z ##[endgroup]
2025-02-25T03:23:46.3053560Z ============================= test session starts ==============================
2025-02-25T03:23:46.3055480Z platform darwin -- Python 3.12.4, pytest-8.2.2, pluggy-1.5.0
2025-02-25T03:23:46.3056470Z rootdir: /Users/runner/work/tardis/tardis
2025-02-25T03:23:46.3056980Z configfile: pyproject.toml
2025-02-25T03:23:46.3057330Z plugins: cov-5.0.0, html-4.1.1, metadata-3.1.1, doctestplus-1.2.1, anyio-4.4.0
2025-02-25T03:23:46.3058300Z collected 1787 items / 1 error / 2 deselected / 1785 selected
2025-02-25T03:23:46.3058510Z 
2025-02-25T03:23:46.3058660Z ==================================== ERRORS ====================================
2025-02-25T03:23:46.3059110Z __ ERROR collecting tardis/energy_input/tests/test_gamma_ray_packet_source.py __
2025-02-25T03:23:46.3059730Z ImportError while importing test module '/Users/runner/work/tardis/tardis/tardis/energy_input/tests/test_gamma_ray_packet_source.py'.
2025-02-25T03:23:46.3060380Z Hint: make sure your test modules/packages have valid Python names.
2025-02-25T03:23:46.3060900Z Traceback:
2025-02-25T03:23:46.3061280Z ../../../micromamba/envs/tardis/lib/python3.12/importlib/__init__.py:90: in import_module
2025-02-25T03:23:46.3061710Z     return _bootstrap._gcd_import(name[level:], package, level)
2025-02-25T03:23:46.3062170Z tardis/energy_input/tests/test_gamma_ray_packet_source.py:4: in <module>
2025-02-25T03:23:46.3062680Z     from tardis.energy_input.gamma_ray_packet_source import RadioactivePacketSource
2025-02-25T03:23:46.3063490Z E   ImportError: cannot import name 'RadioactivePacketSource' from 'tardis.energy_input.gamma_ray_packet_source' (/Users/runner/work/tardis/tardis/tardis/energy_input/gamma_ray_packet_source.py)
2025-02-25T03:23:46.3064280Z =============================== warnings summary ===============================
2025-02-25T03:23:46.3064600Z tardis/io/configuration/config_validator.py:6
2025-02-25T03:23:46.3064970Z tardis/io/configuration/config_validator.py:6
2025-02-25T03:23:46.3065460Z   /Users/runner/work/tardis/tardis/tardis/io/configuration/config_validator.py:6: DeprecationWarning:
2025-02-25T03:23:46.3068220Z   
2025-02-25T03:23:46.3069560Z   jsonschema.RefResolver is deprecated as of v4.18.0, in favor of the https://github.com/python-jsonschema/referencing library, which provides more compliant referencing behavior as well as more flexible APIs for customization. A future release will remove RefResolver. Please file a feature request (on referencing) if you are missing an API for the kind of customization you need.
2025-02-25T03:23:46.3070820Z 
2025-02-25T03:23:46.3071230Z ../../../micromamba/envs/tardis/lib/python3.12/site-packages/jupyter_client/connect.py:22
2025-02-25T03:23:46.3071990Z   /Users/runner/micromamba/envs/tardis/lib/python3.12/site-packages/jupyter_client/connect.py:22: DeprecationWarning:
2025-02-25T03:23:46.3072680Z   
2025-02-25T03:23:46.3073070Z   Jupyter is migrating its paths to use standard platformdirs
2025-02-25T03:23:46.3073700Z   given by the platformdirs library.  To remove this warning and
2025-02-25T03:23:46.3075630Z   see the appropriate new directories, set the environment variable
2025-02-25T03:23:46.3076150Z   `JUPYTER_PLATFORM_DIRS=1` and then run `jupyter --paths`.
2025-02-25T03:23:46.3076810Z   The use of platformdirs will be the default in `jupyter_core` v6
2025-02-25T03:23:46.3077020Z 
2025-02-25T03:23:46.3077170Z tardis/plasma/properties/__init__.py:17
2025-02-25T03:23:46.3078200Z   /Users/runner/work/tardis/tardis/tardis/plasma/properties/__init__.py:17: DeprecationWarning:
2025-02-25T03:23:46.3078650Z   
2025-02-25T03:23:46.3079310Z   The numpy.linalg.linalg has been made private and renamed to numpy.linalg._linalg. All public functions exported by it are available from numpy.linalg. Please use numpy.linalg.LinAlgError instead.
2025-02-25T03:23:46.3079930Z 
2025-02-25T03:23:46.3080110Z -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
2025-02-25T03:23:46.3080460Z 
2025-02-25T03:23:46.3080680Z ---------- coverage: platform darwin, python 3.12.4-final-0 ----------
2025-02-25T03:23:46.3081080Z Coverage HTML written to dir coverage_html_report
2025-02-25T03:23:46.3081500Z Coverage XML written to file coverage.xml
2025-02-25T03:23:46.3081720Z 
2025-02-25T03:23:46.3081900Z =========================== short test summary info ============================
2025-02-25T03:23:46.3082270Z ERROR tardis/energy_input/tests/test_gamma_ray_packet_source.py
2025-02-25T03:23:46.3082740Z !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
2025-02-25T03:23:46.3083140Z ================== 2 deselected, 4 warnings, 1 error in 8.96s ==================
2025-02-25T03:23:47.2515050Z ##[error]Process completed with exit code 2.
